src/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component, PureComponent } from 'react';
+import { render } from 'react-dom';
+
+const HelloOne = () =>
+  React.createElement('h1', {}, 'Hello from create Element');
+
+const HelloTwo = () => <h1>Hello from functional component</h1>;
+
+class HelloThree extends Component {
+  constructor() {
+    super();
+    this.state = {
+      counter: 0
+    };
+  }
+
+  render() {
+    return <h1>Hello from component</h1>;
+  }
+}
+
+class HelloFour extends PureComponent {
+  constructor() {
```

Comment:
There is no need to create state for this component, because we're not using it.